src/utils/MaxwellBoltzmannDistribution.h

```diff
@@ -17,7 +17,7 @@
  * @param dimensions Number of dimensions for which the velocity vector shall be generated. Set this to 2 or 3.
  * @return Array containing the generated velocity vector.
  */
-std::array<double, 3> maxwellBoltzmannDistributedVelocity(double averageVelocity, size_t dimensions) {
+inline std::array<double, 3> maxwellBoltzmannDistributedVelocity(double averageVelocity, size_t dimensions) {
```

Comment:
you cannot force inlining of functions, while in theory it is totally fine to add it everywhere, in practice you want to do it for smaller functions (eg. getters and setters, or other simple functions). Note that compiler might very well just ignore this. Read more [here](https://www.geeksforgeeks.org/inline-functions-cpp/).